submission/submitter/src/main/kotlin/ac/uk/ebi/biostd/submission/submitter/request/SubmissionRequestCleaner.kt

```diff
@@ -28,7 +30,6 @@ class SubmissionRequestCleaner(
         if (current != null) {
             logger.info { "${new.accNo} ${new.owner} Started cleaning common files of version ${new.version}" }
             deleteCommonFiles(new, current)
-            storageService.deleteFtpLinks(current)
```

Comment:
previos changes deleted all ftp folder resulting in files unavaialbiltiy even when not need it.